spec/factories/user_picture_phenotypes.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :user_picture_phenotype do
+    variation "pink"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.